src/ring_theory/witt_vector/is_poly.lean

```diff
@@ -617,37 +617,4 @@ attribute [ghost_simps]
       nat.succ_ne_zero nat.add_sub_cancel nat.succ_eq_add_one
       if_true eq_self_iff_true if_false forall_true_iff forall_2_true_iff forall_3_true_iff
 
-/-!
-### Subtraction of Witt vectors
-
-In Lean, subtraction in a ring is by definition equal to `x + -y`.
-For Witt vectors, this means that subtraction is not defined in terms of
-the polynomials `witt_sub p`.
-
-As a demonstration of some of the techniques developed in this file,
-we show by a computation that evaluating `witt_sub p` on the coefficients of `x` and `y`
-gives the coefficients of `x - y`.
-
-For a more powerful demonstration, see `ring_theory/witt_vector/identities.lean`.
--/
-
-lemma sub_eq (x y : 𝕎 R) :
-  x - y = eval (witt_sub p) ![x, y] :=
```

Comment:
Should there be a proof that `x + -y = eval (witt_sub p) ![x, y]` now that the definition flipped? Or does that exist somewhere else already?

Reply:
Just use `sub_eq_add_neg` and the definition of subtraction. I don't think it's worth having it as separate lemma.

Reply:
Are you saying the proof is trivial, or just that its not useful?

Reply:
Ah, I guess what I'm asking is where the proof of `sub_eq_add_neg` for `𝕎 R` is, assuming its not definitionally true.

Reply:
It should be trivial.